[2022-02-09 09:25:55,211] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXii_HW_data_ingestion_gcs_dag.bigquery_external_table_task scheduled__2019-02-01T23:00:00+00:00 [queued]>
[2022-02-09 09:25:55,223] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXii_HW_data_ingestion_gcs_dag.bigquery_external_table_task scheduled__2019-02-01T23:00:00+00:00 [queued]>
[2022-02-09 09:25:55,224] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:25:55,224] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:25:55,224] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:25:55,237] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2019-02-01 23:00:00+00:00
[2022-02-09 09:25:55,245] {standard_task_runner.py:52} INFO - Started process 396 to run task
[2022-02-09 09:25:55,252] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXii_HW_data_ingestion_gcs_dag', 'bigquery_external_table_task', 'scheduled__2019-02-01T23:00:00+00:00', '--job-id', '655', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXii_data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp0cb3ab38', '--error-file', '/tmp/tmpbv2sm10w']
[2022-02-09 09:25:55,253] {standard_task_runner.py:77} INFO - Job 655: Subtask bigquery_external_table_task
[2022-02-09 09:25:55,328] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXii_HW_data_ingestion_gcs_dag.bigquery_external_table_task scheduled__2019-02-01T23:00:00+00:00 [running]> on host 0092064357b4
[2022-02-09 09:25:55,373] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:25:55,400] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXii_HW_data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2019-02-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-01T23:00:00+00:00
[2022-02-09 09:25:55,402] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 09:25:56,968] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXii_HW_data_ingestion_gcs_dag, task_id=bigquery_external_table_task, execution_date=20190201T230000, start_date=20220209T092555, end_date=20220209T092556
[2022-02-09 09:25:57,034] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:25:57,070] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
